### ISSUE TITLE
arc_summary -d error: TypeError: a bytes-like object is required, not 'str'

### ISSUE THREAD

**Issue by loop-evgeny** [openzfs/zfs#12457]
### System information

Type | Version/Name
 --- | ---
Distribution Name	| Ubuntu
Distribution Version	| 18.04
Kernel Version	| 4.15.0-151-generic
Architecture	| x64
OpenZFS Version	|  Unknown

```
# zfs version
unrecognized command 'version'
```

Python 3.6.9

### Describe the problem you're observing

`arc_summary -d` reports the following error at the end of the normal `arc_summary` output:

```
...
DMU Prefetch Efficiency:					1.94k
	Hit Ratio:			0.26%	5
	Miss Ratio:			99.74%	1.94k

Traceback (most recent call last):
  File "/usr/sbin/arc_summary", line 991, in <module>
    main()
  File "/usr/sbin/arc_summary", line 986, in main
    page(Kstat)
  File "/usr/sbin/arc_summary", line 865, in _tunable_summary
    description_list = p.communicate()[0].strip().split('\0')
TypeError: a bytes-like object is required, not 'str'
```

### Describe how to reproduce the problem

Run `arc_summary -d`

**Comment by gdevenyi** [comment]
Sounds like a python2/python3 problem.

**Comment by rincebrain** [comment]
#6849 

Ubuntu 18.04 ships a highly-patched 0.7.5, which is from December 2017. This was [patched in master in November 2017](https://github.com/openzfs/zfs/commit/5277f208f290ea4e2204800a66c3ba20a03fe503) and [in 0.7.x on January 30th](https://github.com/openzfs/zfs/commit/7de8fb33a206fabb06261c0a5b79656d66efb32f).

I suggest you ask Ubuntu to cherrypick it or choose to run a newer version.

**Comment by loop-evgeny** [comment]
Thanks, I did not find that issue when I searched for "is:issue arc_summary TypeError". Turns out you have to search for "is:issue arc_summary.py TypeError". This is a duplicate, then, closing.
